orchagent/flexcounterorch.cpp

```diff
@@ -70,6 +70,10 @@ void FlexCounterOrch::doTask(Consumer &consumer)
                 }
                 else if(field == FLEX_COUNTER_STATUS_FIELD)
                 {
+                    // Currently the counters are disabled by default
+                    // The queue maps will be generated as soon as counters are enabled
+                    gPortsOrch->generateQueueMap();
```

Comment:
Is the map generated after queue counters are enabled or all the counters are enabled?

Reply:
The map is generated after activation of the flex counters. As I understand the maps are related to port queues. So they will be used with queue counters.